dist/video-stream-merger.js

```diff
@@ -19,14 +30,8 @@ function VideoStreamMerger (opts) {
   self._canvas.setAttribute('width', self.width)
   self._canvas.setAttribute('height', self.height)
   self._canvas.setAttribute('style', 'position:fixed; left: 110%; pointer-events: none') // Push off screen
-  document.body.appendChild(self._canvas)
   self._ctx = self._canvas.getContext('2d')
 
-  // Hidden div to contain video elements
-  self._container = document.createElement('div')
```

Comment:
Since a hidden div doesn't seem to be needed, could you also remove the reference to `self._container` in `removeStream`